src/application/Application.mjs

```diff
@@ -284,6 +387,12 @@ export default class Application extends Component {
         this.stage.setApplication(undefined);
         this._updateAttachedFlag();
         this._updateEnabledFlag();
+
+        if (this.__keypressTimers) {
```

Comment:
Since `this.__keypressTimers` is a Map object created in constructor and never reassigned, this will always yield true. You may also want to call `this._keypressTimers.clear()` to get rid of dangling references.

Reply:
valid point, changed the condition to validate map size and clear the map